convert_numeric

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+#
+#                         License
+#
+# Copyright (C) 2024  David Valin dvalin@redhat.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Convert various time and memory units.
+
+usage()
+{
+	echo "usage $1 <unit>"
+	echo "-h --help --usage: This help message"
+	echo "--cnv_unit <u>: Unit to convert to."
```

Comment:
Maybe use `--to-unit` and `--from-unit` here, makes it more clear what the script will be doing.